packages/filesystem/src/common/filesystem.ts

```diff
@@ -119,6 +119,45 @@ export interface FileSystem extends JsonRpcServer<FileSystemClient> {
      */
     getDrives(): Promise<string[]>;
 
+    /**
+     * Tests a user's permissions for the file or directory specified by URI.
+     * The mode argument is an optional integer that specifies the accessibility checks to be performed.
+     * Check `FileAccess.Constants` for possible values of mode.
+     * It is possible to create a mask consisting of the bitwise `OR` of two or more values (e.g. FileAccess.Constants.W_OK | FileAccess.Constants.R_OK).
+     * If `mode` is not defined, `FileAccess.Constants.F_OK` will be used instead.
+     */
+    access(uri: string, mode?: number): Promise<boolean>
+
+}
+
+export namespace FileAccess {
+
+    export namespace Constants {
+
+        /**
+         * Flag indicating that the file is visible to the calling process.
+         * This is useful for determining if a file exists, but says nothing about rwx permissions. Default if no mode is specified.
+         */
+        export const F_OK: number = 0;
+
+        /**
+         * Flag indicating that the file can be read by the calling process.
+         */
+        export const R_OK: number = 4;
```

Comment:
Minor, but it would be nice to have it defined in the order of the bitwise, i.e. 1, 2,4,..

Reply:
The order is [from Node.js](https://nodejs.org/dist/latest-v8.x/docs/api/fs.html#fs_fs_access_path_mode_callback)